package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jitsi-meet-electron-utils",
-  "version": "0.6.8",
+  "version": "0.6.9",
```

Comment:
Make it 1.0 already!

Reply:
We should! But let's wait for the next minor bump at least?